backend/src/core/routes/index.ts

```diff
@@ -78,6 +79,7 @@ const router = Router()
 router.use('/ping', ping)
 router.use('/auth', authenticationRoutes)
 router.use('/stats', statsRoutes)
+router.use('/protect', protectedCampaignRoutes)
```

Comment:
hmmm are we doing `/protect` or `/p`? Could we use `config.get('protectedPath')`?